### PR TITLE
DepthAI SDK 1.11.0

### PR DESCRIPTION
**Pull request by daniilpastukhov**
- Address issues with encoded NN output.
- Added auto IR mode.
- `NNComponent.config_tracker()` now provides `calculate_speed` argument to enable speed on-host calculation.
- Added pointcloud support.

### CODE REVIEW

File: depthai_sdk/examples/NNComponent/two_stage_detection.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from depthai_sdk import OakCamera
```
Comment (Erol444) [inline depthai_sdk/examples/NNComponent/two_stage_detection.py:1]: still WIP

File: depthai_sdk/src/depthai_sdk/components/pointcloud_component.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import logging
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/components/pointcloud_component.py:1]: TODO: add docs for `PointcloudComponent`

File: depthai_sdk/src/depthai_sdk/nn_models/facial_landmarks_68_160x160/config.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "model": {
+        "model_name":"facial_landmarks_68_160x160",
+        "zoo": "depthai"
+    },
+    "nn_config": {
+        "output_format" : "raw"
+    },
+    "handler": "handler.py",
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/nn_models/facial_landmarks_68_160x160/config.json:9]: 2022.1 doesn't work afaik. todo: test
```suggestion
    "handler": "handler.py",
    "openvino_version": "2021.4",
```

**Review by Erol444 — CHANGES_REQUESTED** [review]

**Comment by daniilpastukhov** [comment]
@Erol444 is anything missing right now? If not, we can merge and release 1.11 :-)

**Pull request merged**